packages/content-management/favorites/favorites_server/src/favorites_service.ts

```diff
@@ -7,9 +7,19 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
+// eslint-disable-next-line max-classes-per-file
 import type { SavedObject, SavedObjectsClientContract } from '@kbn/core-saved-objects-api-server';
 import { Logger, SavedObjectsErrorHelpers } from '@kbn/core/server';
 import { favoritesSavedObjectType, FavoritesSavedObjectAttributes } from './favorites_saved_object';
+import { FavoritesRegistry } from './favorites_registry';
+
+export interface FavoritesState {
+  favoriteIds: string[];
+  favoriteMetadata?: Record<string, object>;
+}
+
+// Limit the number of favorites to prevent too large objects due to metadata
+const FAVORITES_LIMIT = 100;
```

Comment:
@Dosant I would like to export this to reuse it in the client side on my package. Now it is located to the server side directory. I want to to export it and reuse it from the favorites_public directory. Not sure how to do it with the current structure though.

Reply:
Got it, will take a look